.github/workflows/release.yml

```
@@ -74,29 +73,35 @@ jobs:
         run: pip install -r requirements.txt
 
       - name: Install pyinstaller
-        run: pip install pyinstaller
+        run: pip install pyinstaller==5.0.1
 
       - name: Build
+        if: matrix.os != 'macos-10.15'
         working-directory: ${{ env.MCUBOOT_PATH }}/scripts/
         run: pyinstaller --onefile main.py -n ${{ env.PROJECT_NAME }}
 
+      - name: Build macOS
+        if: matrix.os == 'macos-10.15'
```

```suggestion
        if: runner.os == 'macOS'
        working-directory: ${{ env.MCUBOOT_PATH }}/scripts/
        run: pyinstaller --onefile main.py -n ${{ env.PROJECT_NAME }}

      - name: Build macOS
        if: runner.os == 'macOS'
```
Here is a possible improvement for maintenance of the workflow. The reasoning is that otherwise you need to remember to change these lines any time the runner version is updated.

I did something similar in the "Test Go" workflow: https://github.com/arduino/tooling-project-assets/pull/142

If these are changed to use `runner.os`, it would be best to do the same for the other `'windows-latest'` conditional steps as well. The conditionals for the "latest" runner version are probably less prone to becoming outdated, but the workflow would be confusing if it was using `matrix.os` for identifying one OS flavor and `runner.os` for the other.

---

smart! I did not knew about this